### ISSUE TITLE
Hitboxes don't really align with the zombies animations

### ISSUE THREAD

**Issue by Paindeluxe** [nzp-team/nzportable#181]
**To Reproduce**
_Steps to reproduce the behavior:_
1. Go find a map with the zombie spawnflag of 6.
2. Shoot at zombies who are currently Raising.
3. Try shooting slightly above next time.
4. See Error.

**Expected behavior**
_A clear and concise description of what you expected to happen._

The hitboxes align well enough with the model's animation

**Screenshots/Footage**
_If applicable, add screenshots or video to help explain your problem._

https://youtu.be/3RJ5lpB_62g

**Affected Platforms:**

PSP

# Please make note of any platforms you DID NOT test on.

- PS VITA
- Nintendo Switch
- 3DS

**Additional context**
_Add any other context about the problem here._

I appreciate the extra fps (:

**Comment by BCDeshiG** [comment]
Yeah this has been around for a while, also present on PC at the very least.
It's probably because the zombie has a standing hitbox when it spawns in.
